src/Stratis.Bitcoin.Features.PoA/PoAFeature.cs

```diff
@@ -89,9 +89,6 @@ public override Task InitializeAsync()
 
             this.ReplaceBlockStoreBehavior(connectionParameters);
 
-            this.federationManager.Initialize();
```

Comment:
I think this is wrong as if we dont initialize the federation members first then the voting manager doesnt have any members to interrogate

Reply:
This appears to be a bit of a chicken egg scenario?